app/models/manageiq/providers/ibm_power_hmc/infra_manager/vm.rb

```diff
@@ -7,6 +7,10 @@ class ManageIQ::Providers::IbmPowerHmc::InfraManager::Vm < ManageIQ::Providers::
     unsupported_reason_add(:control, _("Host is not HMC-managed")) unless host_hmc_managed
   end
 
+  supports :rename do
+    unsupported_reason_add(:rename, _("Host is not HMC-managed")) unless host.hmc_managed
```

Comment:
Above you use `host_hmc_managed` and here you use `host.hmc_managed` I think we should be consistent and probably prefer using the delegate (if host is `nil` for example this will fail and that will work)
```suggestion
    unsupported_reason_add(:rename, _("Host is not HMC-managed")) unless host_hmc_managed
```

Reply:
Yeah that makes sense, good catch!